broken-links-script/Extractlinks.js

```diff
@@ -63,34 +69,35 @@ const resolveFullUrl = (link, relativePath, fileContent) => {
   }
 
   if (link.startsWith("#")) {
-    const fileDir = path.dirname(relativePath);
+    const fileDir = path.dirname(relativePath).replace(/\\/g, "/");
```

Comment:
If you need to convert disk path (platfom specific) into url path, you could do:
```suggestion
    const fileDir = path.dirname(relativePath).replaceAll(path.sep, '/');
```